archunit/src/main/java/com/tngtech/archunit/core/domain/JavaCodeUnit.java

```diff
@@ -69,6 +72,11 @@ public JavaClassList getParameters() {
         return new JavaClassList(parameters);
     }
 
+    @Override
+    public JavaClassList getThrows() {
```

Comment:
I would call this `getThrowsDeclarations()` and return `ThrowsDeclarations implements Iterable<ThrowsDeclaration>` (compare statement above)